pkg/ingester/transfer.go

```diff
@@ -38,9 +38,9 @@ var (
 		Name: "cortex_ingester_sent_files",
 		Help: "The total number of files sent by this ingester whilst leaving.",
 	})
-	receivedFiles = prometheus.NewCounter(prometheus.CounterOpts{
-		Name: "cortex_ingester_received_files",
-		Help: "The total number of files received by this ingester whilst joining",
+	receivedBytes = prometheus.NewCounter(prometheus.CounterOpts{
```

Comment:
I'm not much convinced about this change, because of two reasons:
1. Drift between `cortex_ingester_sent_files` and `cortex_ingester_received_bytes `
2. Drift between chunks storage (`cortex_ingester_sent_chunks` and `cortex_ingester_received_chunks`) which count chunks and TSDB which count bytes

I would suggest to rollback to count files (and fix the count).

Reply:
File count is kinda weird though since they're variable sized right? Chunks are a static size so one can infer how much data was actually sent. I guess we could count number of batches sent? I guess my question is what type of info are we trying to glean here? I would imagine it would be much more useful to see how much data is being sent on xfer, vs number of arbitrarily sized files.

Reply:
Given we're in doubt and given this metric cardinality is low, why don't we keep the current one and we add bytes as well?

Reply:
Should the former one be batches transferred or files. If the latter we have to do some tracking of filenames to determine the correct file count.

Reply:
I would suggest to count files and bytes. The files count should be incremented only when we create a new file in `TransferTSDB()` (should just be a matter of placing the counter increment in the right place, no extra logic should be required).

Reply:
Good call. Done